GROMACS/Methane/run.sh

```diff
@@ -68,60 +69,54 @@ xtp_run -e eanalyze -o OPTIONFILES/eanalyze.xml -f state.hdf5
-cp $VOTCASHARE/xtp/packages/xtpdft.xml OPTIONFILES/
-cp $VOTCASHARE/xtp/packages/esp2multipole.xml OPTIONFILES/
-
-changeoption dftpackage OPTIONFILES/xtpdft.xml OPTIONFILES/eqm.xml
-changeoption gwbse_options OPTIONFILES/gwbse.xml OPTIONFILES/eqm.xml
-
+cp "$VOTCASHARE/xtp/xml/eqm.xml" OPTIONFILES/
 changeoption ranges full OPTIONFILES/gwbse.xml
+changeoption map_file system.xml OPTIONFILES/eqm.xml
 
 xtp_parallel -e eqm -o OPTIONFILES/eqm.xml -f state.hdf5 -s 0 -j "write"
 sed -i "s/AVAILABLE/COMPLETE/g" eqm.jobs
 sed -i '0,/COMPLETE/s/COMPLETE/AVAILABLE/' eqm.jobs
 sed -i '0,/COMPLETE/s/COMPLETE/AVAILABLE/' eqm.jobs
 sed -i '0,/COMPLETE/s/COMPLETE/AVAILABLE/' eqm.jobs
 
-xtp_parallel -e eqm -o OPTIONFILES/eqm.xml -f state.hdf5 -s 0 -j run -c 1 -t 1
+xtp_parallel -e eqm -o OPTIONFILES/eqm.xml -f state.hdf5 -s 0 -j run -c 1 -t 4
 
 #running iqm
 #iqm runs qm calculations for each pair in the hdf5 file, it consists of three stages first writing a jobfile, then running the calculations, if necessary 
 # on multiple machines and then reading them into the .hdf5 file
 echo "Running iQM"
 
-cp $VOTCASHARE/xtp/xml/iqm.xml OPTIONFILES/
-cp $VOTCASHARE/xtp/packages/gwbse.xml OPTIONFILES/gwbse_pair.xml
-cp $VOTCASHARE/xtp/packages/xtpdft.xml OPTIONFILES/xtpdft_pair.xml
-cp $VOTCASHARE/xtp/packages/bsecoupling.xml OPTIONFILES/
-
-changeoption bsecoupling_options OPTIONFILES/bsecoupling.xml OPTIONFILES/iqm.xml
-changeoption dftpackage OPTIONFILES/xtpdft_pair.xml OPTIONFILES/iqm.xml
-changeoption gwbse_options OPTIONFILES/gwbse_pair.xml OPTIONFILES/iqm.xml
-changeoption read_guess 1 OPTIONFILES/xtpdft_pair.xml
-changeoption energy 1e-2 OPTIONFILES/xtpdft_pair.xml
+cp "$VOTCASHARE/xtp/xml/iqm.xml" OPTIONFILES/
+changeoption map_file system.xml OPTIONFILES/iqm.xml
+changeoption ranges full OPTIONFILES/iqm.xml
 
-changeoption ranges full OPTIONFILES/gwbse_pair.xml
+# Append the states to read to iqm.xml
+TAIL=$(tail -n 2 OPTIONFILES/iqm.xml)
+head -n -2 OPTIONFILES/iqm.xml > OPTIONFILES/tmp
 
-changeoption singlet "Methane:s1" OPTIONFILES/iqm.xml
-changeoption triplet "Methane:t1" OPTIONFILES/iqm.xml
-changeoption electron "Methane:e1" OPTIONFILES/iqm.xml
-changeoption hole "Methane:h1" OPTIONFILES/iqm.xml
+cat >> OPTIONFILES/tmp <<- EOM
+<readjobfile help="which states to read into the jobfile for each segment type">
+     <singlet>Methane:s1</singlet>
+     <triplet>Methane:t1</triplet>
+     <electron>Methane:e1</electron>
+     <hole>Methane:h1</hole>
+</readjobfile>
+EOM
 
-changeoption tasks "GW" OPTIONFILES/gwbse_pair.xml
+echo "${TAIL}" >> OPTIONFILES/tmp
+mv OPTIONFILES/tmp  OPTIONFILES/iqm.xml
 
 xtp_parallel -e iqm -o OPTIONFILES/iqm.xml -f state.hdf5 -s 0 -j "write"
 sed -i "s/AVAILABLE/COMPLETE/g" iqm.jobs
 sed -i '0,/COMPLETE/s/COMPLETE/AVAILABLE/' iqm.jobs
 
-xtp_parallel -e iqm -o OPTIONFILES/iqm.xml -f state.hdf5 -s 0 -j run -c 1 -t 1
+xtp_parallel -e iqm -o OPTIONFILES/iqm.xml -f state.hdf5 -s 0 -j run -c 1 -t 4
```

Comment:
why do you want to default to 4?

Reply:
I put it for testing. Shall I changed all threads to 1?